src/components/LessonTitle/LessonTitle.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { LessonTitle } from "./LessonTitle";
+
+export default {
+    title: 'Components/Lesson Title',
+    component: LessonTitle,
+  } as ComponentMeta<typeof LessonTitle>;
+  
+  const Template: ComponentStory<typeof LessonTitle> = (args) => (
+    <LessonTitle {...args} />
+  );
+
+  export const MainLessonTitle = Template.bind({});
+  MainLessonTitle.args = {
+    surahNameEn: 'Al-’Adiyat',
+    surahNameTranslation: 'The Courser',
+    surahNameAr: 'سُوۡرَةُ العَادیَات',
+    surahVerses: '11',
+    introSections: ['Name', 'Revelation', 'Theme'],
+    buttonLabel: 'Jump to Lecture'
+  }
+
+  export const InnerLessonTitle = Template.bind({});
```

Comment:
Maybe add a comment describing the difference between the titles:
i.e. `// Inner lesson title is the lesson card that appears in the surah intro page`